SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -192,7 +196,13 @@ class CSourceBase_TurbSA : public CNumerics {
       }
 
       /*--- Compute ft2 term ---*/
-      ft2::get(var);
+      if(transition_AFT) {
+        var.lnintermittecncy = exp(intermittency_i);
```

Comment:
it's not lnintermittecncy because you take exp of it right? Can you use the `intermittency` that is already in the variables?